conductr_cli/sandbox_main.py

```diff
@@ -168,11 +168,6 @@ def build_parser():
                             dest='no_default_features',
                             default=False,
                             action='store_true')
-    run_parser.add_argument('--no-wait',
```

Comment:
I guess this has been removed because now we need to wait until ConductR has been started to do the additional validation.

Note that this also affects ConductR 2.0 and 1.x for where we do not perform the license logic. For me, that is fine.